chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -136,6 +142,9 @@ def simplified_dropconnect(x, W, b=None, ratio=.5, train=True, mask=None):
             Otherwise, The mask must be ``(n, M, N)`` shaped array.
```

Comment:
OK,
I think again, we had better check mask shape.

And I added test for mask shape.